css/views.css

```diff
@@ -838,11 +838,51 @@ table.runs td.comment {
 */
 .lockfield {
    background-image: url('../../../plugins/fusioninventory/pics/lock.png');
-   background-repeat: no-repeat; 
+   background-repeat: no-repeat;
    background-position: right;
 }
 
 .copyleft {
   display:inline-block;
   transform: rotate(180deg);
-}
\ No newline at end of file
+}
+
+.fi_board {
+   width: 1300px;
```

Comment:
Why this value? It's not compliant with "small" screens and its breaks responsiveness, does'nt it?